src/components/app/app.js

```diff
@@ -35,23 +40,60 @@ export class AppComponent {
       onAboutClick: this.handleAboutOnclick.bind(this)
     });
     this.toolBarComponent.mount();
-    this.drawerComponent = new DrawerComponent(this.drawerMountPoint, {
-      onAddAccountClick: this.handleAddAccountClick.bind(this),
-      onAddTagClick: this.handleAddTagOnclick.bind(this)
+    this.addAccountDialog = new AddAccountComponent(this.addAccountMountPoint, {
```

Comment:
because you have your account component maybe it is better to move add account dialog from here?